### ISSUE TITLE
Accuracy of the output

### ISSUE THREAD

**Issue by Lilzhuzixi** [ShuvenduRoy/CoPrompt#6]
Dear author, I am a research novice. 
I encountered some issues while reproducing the results of the paper, which resulted in the accuracy of the output not meeting the requirements of the paper. I don't know if there are any areas that need to be modified in the given code? Here is the config file I have modified based on your paper. Please take a look. Are there any other files that need to be modified?

Looking forward to your reply. Thank you very much！

`DATALOADER:
  TRAIN_X:
    BATCH_SIZE: 4
  TEST:
    BATCH_SIZE: 100
  NUM_WORKERS: 8
  K_TRANSFORMS: 2

INPUT:
  SIZE: (224, 224)
  INTERPOLATION: "bicubic"
  PIXEL_MEAN: [0.48145466, 0.4578275, 0.40821073]
  PIXEL_STD: [0.26862954, 0.26130258, 0.27577711]
  TRANSFORMS: ["randaugment", "normalize"]

OPTIM:
  NAME: "sgd"
  LR: 0.035
  MAX_EPOCH: 8
  LR_SCHEDULER: "cosine"
  WARMUP_EPOCH: 1
  WARMUP_TYPE: "constant"
  WARMUP_CONS_LR: 1e-5

TRAIN:
  PRINT_FREQ: 20

MODEL:
  BACKBONE:
    NAME: "ViT-B/16"

TRAINER:
  CoPrompt:
    N_CTX: 2
    CTX_INIT: "a photo of a"
    PREC: "fp16"
    PROMPT_DEPTH: 12
  W: 8.0
  DISTILL: "cosine"
`
